### PR TITLE
Add Kubernetes probes stubs

### LINKED ISSUE #4306: Provide Kubernetes probes
# What should be done?

What documentation in #4424 says.

# DoD

- Implement probes as documentation says.

### PR DESCRIPTION
**Pull request by noisersup**
# Description

Closes #4306.

## Readiness checklist

- [ ] I added/updated unit tests (and they pass).
- [ ] I added/updated integration/compatibility tests (and they pass).
- [ ] I added/updated comments and checked rendering.
- [ ] I made spot refactorings.
- [ ] I updated user documentation.
- [ ] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Milestone (`Next`), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

**Review by rumyantseva — APPROVED** [review]
Looks good for the beginning!

**Comment by noisersup** [comment]
@rumyantseva PTAL, I've removed one TODO comment, as I believe that's what we expect from liveness implementation.

File: internal/util/debug/debug.go [hunk 1]
```diff
@@ -56,11 +56,32 @@ func RunHandler(ctx context.Context, addr string, r prometheus.Registerer, l *za
 	}
 	must.NoError(statsviz.Register(http.DefaultServeMux, opts...))
 
+	http.HandleFunc("/debug/started", func(r http.ResponseWriter, _ *http.Request) {
+		// TODO https://github.com/FerretDB/FerretDB/issues/4306
+		r.WriteHeader(http.StatusOK)
+	})
+
+	// healthz handler, which is used for liveness probe, returns StatusOK when reached.
+	// This ensures that listener is running.
```
Comment (rumyantseva) [inline internal/util/debug/debug.go:65]: I wouldn't say it proactively ensures that the listener is running, but when the listener is stopped, everything stops, so it's enough :) 

We can improve the comments later, for the first PR this looks enough to me.

**Review by rumyantseva — DISMISSED** [review]

**Review by chilagrow — APPROVED** [review]

File: internal/util/debug/debug.go [hunk 2]
```diff
@@ -56,11 +56,32 @@ func RunHandler(ctx context.Context, addr string, r prometheus.Registerer, l *za
 	}
 	must.NoError(statsviz.Register(http.DefaultServeMux, opts...))
 
+	http.HandleFunc("/debug/started", func(r http.ResponseWriter, _ *http.Request) {
```
Comment (AlekSi) [inline internal/util/debug/debug.go:59]: We want to know how those handlers are called. We should instrument them with promhttp

File: internal/util/debug/debug.go [hunk 3]
```diff
@@ -56,11 +62,59 @@ func RunHandler(ctx context.Context, addr string, r prometheus.Registerer, l *za
 	}
 	must.NoError(statsviz.Register(http.DefaultServeMux, opts...))
 
+	requestCount := prometheus.NewCounterVec(
```
Comment (rumyantseva) [inline internal/util/debug/debug.go:65]: Have you tested this?

I don't see where we increment this counter. Also, `collect` and `decribe` seem missing. 

And also, I think @AlekSi meant we could use this https://pkg.go.dev/github.com/prometheus/client_golang/prometheus/promhttp library.

**Review by AlekSi — APPROVED** [review]

**Review by rumyantseva — APPROVED** [review]
Ok, I tried it locally :) 
Looks good!

**Review by chilagrow — APPROVED** [review]

**Pull request merged**